addon/helpers/excerpt.js

```diff
@@ -6,11 +6,7 @@ import downsize from 'downsize-cjs';
 export function getExcerpt(html, truncateOptions) {
   truncateOptions = truncateOptions || {};
   // Strip inline and bottom footnotes
-  var excerpt = html.replace(/<a href="#fn.*?rel="footnote">.*?<\/a>/gi, '');
-  excerpt = excerpt.replace(/<div class="footnotes"><ol>.*?<\/ol><\/div>/, '');
-  // Strip other html
-  excerpt = excerpt.replace(/<\/?[^>]+>/gi, '');
-  excerpt = excerpt.replace(/(\r\n|\n|\r)+/gm, ' ');
+  var excerpt = stripHTML(html);
```

Comment:
hmmmmmm var

Reply:
omg 🙈  I am pretty sure this came from a copy pasta. fixed now tho!